ultet/src/main/scala/.gitkeep

```diff
@@ -1 +0,0 @@
-To remove when actual content is present
```

Comment:
what is this file?

Reply:
It was there, just to have the directory in git.